src/BizHawk.Client.Common/RomLoader.cs

```diff
@@ -483,7 +483,7 @@ private IEmulator MakeCoreFromCoreInventory(CoreInventoryParameters cip, string
 						return;
 					}
 
-					if (_config.GbAsSgb)
+					if (_config.PreferredCores[game.System].Contains(CoreNames.Bsnes115))

```

Comment:
This is a bit of an unfortunate hack that also happens to exclude legacy BSNES (even though that is currently not supported anyway). I would prefer to do this explicitly like this:
```csharp
if (_config.PreferredCores[game.System] is CoreNames.Bsnes115 or CoreNames.SubBsnes115)
```